projects/scion/toolkit.internal/widgets/src/tabbar/public_api.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2018-2019 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+export { SciTabbarModule } from './tabbar.module';
+export { SciTabbarComponent } from './tabbar.component';
```

Comment:
You also need to export `SciTabDirective`.

Reply:
done